circuits/cpp/src/aztec3/circuits/rollup/base/c_bind.cpp

```diff
@@ -2,7 +2,7 @@
 #include "aztec3/utils/dummy_composer.hpp"
 #include "index.hpp"
 #include "init.hpp"
-#include "utils.hpp"
+#include "aztec3/circuits/rollup/test_utils/utils.hpp"
```

Comment:
not sure if this is used, i think it was refering to utils for the dummy composer?

Reply:
Addressed in https://github.com/AztecProtocol/aztec3-packages/pull/342/commits/0091cd760999ae8ebc760ef640393658ccb36a51